.pre-commit-config.yaml

```
@@ -11,6 +11,22 @@ repos:
       - id: check-yaml
       - id: debug-statements
       - id: mixed-line-ending
+  - repo: https://github.com/pre-commit/pygrep-hooks
+    rev: v1.10.0
+    hooks:
+      - id: python-check-blanket-noqa
+      - id: python-check-blanket-type-ignore
+      - id: python-check-mock-methods
+      - id: python-no-log-warn
+      # - id: python-use-type-annotations  # too many false positives
+      - id: rst-backticks
+      - id: rst-directive-colons
+      - id: rst-inline-touching-normal
+      - id: text-unicode-replacement-char
+  - repo: https://github.com/executablebooks/mdformat
+    rev: 0.7.18
+    hooks:
+    - id: mdformat
```

apparently, we can fix (some of) the surprising bracket escapes with
```suggestion
    - id: mdformat
      additional_dependencies: [mdformat-gfm]
```

---

though I wonder if instead of using `mdformat` we should rely on `prettier` instead?

---

FWIW in prql we quite successfully use `prettier` to format all non-core code, including yaml / json / md; would recommend: https://github.com/PRQL/prql/blob/86becc591eb1a6421695ccccdf67dd8ab5c775ec/.pre-commit-config.yaml#L18-L22

...and it has a nice VS Code extension (and I guess emacs for the pros among us...)